service-base/src/main/java/org/eclipse/hono/service/management/credentials/PasswordSecret.java

```diff
@@ -12,13 +12,16 @@
  *******************************************************************************/
 package org.eclipse.hono.service.management.credentials;
 
+import io.vertx.core.json.JsonObject;
 import static org.eclipse.hono.util.CredentialsConstants.FIELD_SECRETS_HASH_FUNCTION;
 import static org.eclipse.hono.util.CredentialsConstants.FIELD_SECRETS_PWD_HASH;
+import static org.eclipse.hono.util.CredentialsConstants.FIELD_SECRETS_PWD_PLAIN;
```

Comment:
all these constants should be imported from `RegistryManagementConstants` instead of `CredentialsConstants`